components/util/videoCards.tsx

```diff
@@ -0,0 +1,118 @@
+import { FC } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { AiFillPlayCircle } from "react-icons/ai";
+import { Section } from "./section";
+import { Container } from "./container";
+import Button from "../button/button";
+
+export type VideoCardProps = {
+  link: string,
+  thumbnail: string,
+  title: string,
+};
+
+const VideoCard: FC<VideoCardProps> = ({ link, thumbnail, title }) => {
+  return (
+    <div
+      className="flex flex-col rounded bg-white"
+    >
+      <div>
+        <Image
+          src={thumbnail || "/public/images/ssw-logo.svg"}
+          alt={thumbnail ? title : "SSW Consulting"}
+          height={0}
+          width={500}
+          className="w-full"
+        />
+      </div>
+      <div className="flex h-full items-center justify-between p-5">
+        <span className="w-3/4 text-left text-lg font-bold text-sswRed">{title}</span>
+        <Link href={link || ""}>
+          <AiFillPlayCircle className="text-sswRed" size={50} />
+        </Link>
```

Comment:
I believe the intent is to play the video on the site as opposed to navigating users away from the site

Reply:
As discussed with @BrookJeynes, this was resolved